### PR TITLE
Indent Page

### PR DESCRIPTION
**Pull request by saadmalik200**
We created the Indent page and a separate style sheet (indent.css) for it.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
```
Comment (Lvrks) [inline .vscode/settings.json]: You should delete this file. This happens when you are running more than one live-server

File: common/indents.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Indents</title>
+    <link rel="stylesheet" href="../css/style.css" />
+    <link rel="stylesheet" href="../css/indents.css" />
```
Comment (Lvrks) [inline common/indents.html:9]: Looks like you forgot to add this css file.

**Review by Lvrks — CHANGES_REQUESTED** [review]

**Comment by Lvrks** [comment]
It looks like there are some merge conflicts from mostly moving things around. But also: 

-  I don't think we need any of the extra fonts. 
- I wouldn't add the empty `html` files for pages not being worked on by you

File: common/indents.html (hunk 2 above)
Comment (saadmalik200) [reply common/indents.html:9]: We have fixed the problem and the file is there now.

File: .vscode/settings.json (hunk 1 above)
Comment (saadmalik200) [reply .vscode/settings.json]: Deleted this file. Problem Solved.

**Comment by saadmalik200** [comment]
@HelonDCI Could you please check if everything here is resolved or is there anything else we need to do here.

**Comment by saadmalik200** [comment]
@HelonDCI We completed the history page as well but we forgot to switch branch from indent to history before we did that. That is why it is here on the pull request from the indent page.

**Pull request merged**